internal/bgp/frr/config.go

```diff
@@ -66,7 +69,9 @@ router bgp {{.MyASN}}
   {{- end }}
 {{- if ne .BFDProfile ""}} 
   neighbor {{.Addr}} bfd profile {{.BFDProfile}}
-{{- end -}}
+{{- end }}
+  neighbor {{.Addr}} prefix-list deny-all in
+  neighbor {{.Addr}} prefix-list v6deny-all in
```

Comment:
this should be done inside address-family block where we apply route-map and based on the version u pick one or the other

Reply:
This will make the template uglier, as we'd need to do it both when there are no advertisements and when we have those, plus iff-ing on the ip family.
I think it's fair to say we always block everything.

Reply:
ugly ofcourse but it address-family config, not sure what will happen when u pass v4 address to v6 prefix list and other way around to be safe having them under their respective address-family will be less error prune IMO

Reply:
(I mean, if it works)

Reply:
I think you are right. The resulting configuration renders like this :

```
 !
 address-family ipv4 unicast
  neighbor fc00:f853:ccd:e793::5 activate
  neighbor fc00:f853:ccd:e793::5 prefix-list v6deny-all in
  neighbor fc00:f853:ccd:e793::6 activate
  neighbor fc00:f853:ccd:e793::6 prefix-list v6deny-all in
 exit-address-family
 !
 address-family ipv6 unicast
  neighbor fc00:f853:ccd:e793::5 activate
  neighbor fc00:f853:ccd:e793::6 activate
 exit-address-family
```